pkg/logs/config/config.go

```diff
@@ -63,23 +71,30 @@ func BuildServerConfig() (*ServerConfig, error) {
 		if err != nil {
 			return nil, fmt.Errorf("could not parse logs_dd_url port: %v", err)
 		}
-		return NewServerConfig(
-			host,
-			port,
-			!LogsAgent.GetBool("logs_config.logs_no_ssl"),
-		), nil
+		main.Host = host
+		main.Port = port
+		useSSL = !LogsAgent.GetBool("logs_config.logs_no_ssl")
```

Comment:
I would set UseProto here too, it's very similar to SSL.

Reply:
👍